index.html

```diff
@@ -354,7 +354,7 @@ <h2>The <dfn>PerformanceEntry</dfn> interface</h2>
     <p>When <dfn>toJSON</dfn> is called, run [[WebIDL]]'s <a>default toJSON
     steps</a>.</p>
 
-    <p>A <a>PerformanceEntry</a> has a {{DOMHighResTimeStamp}} <dfn>endTime</dfn>,
+    <p>A <a>PerformanceEntry</a> has a {{DOMHighResTimeStamp}} <dfn export>end time</dfn>,
```

Comment:
This is also used below (line 368, where I can't add a review comment).

Reply:
Fixed